Contents/Code/youtube_dl_helper.py

```diff
@@ -13,6 +13,26 @@
 from typing import Optional
 import youtube_dl
 
+class LoggerProxy(object):
+    def debug(self, msg, *args, **kwargs):
+        Log.Debug(msg, *args, **kwargs)
+
+    def info(self, msg, *args, **kwargs):
+        Log.Info(msg, *args, **kwargs)
+
+    def warning(self, msg, *args, **kwargs):
+        Log.Warn(msg, *args, **kwargs)
+
+    def error(self, msg, *args, **kwargs):
+        Log.Error(msg, *args, **kwargs)
+
+    def critical(self, msg, *args, **kwargs):
+        Log.Critical(msg, *args, **kwargs)
+
+    def exception(self, msg, *args, **kwargs):
+        Log.Exception(msg, exc_info=True, *args, **kwargs)
+
+youtube_logger = LoggerProxy()
```

Comment:
```suggestion
# get the plugin logger
plugin_logger = logging.getLogger(plugin_identifier)
```

Reply:
Oh, well, that's certainly better. This is my first time writing code for Plex plugin-side so I wasn't sure if Plex has its own logging system or if I could use the regular `logging` module. I have a surefire way to trigger an exception; I'll just restart the plugin and let it try downloading the theme for one of the various movies with unavailable videos. I'll update